### ISSUE TITLE
Switch to native browser tooltips

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#4321]
As an accessibility requirement we should drop any usage of the custom v-tooltip directive / Tooltip from `@nextcloud/vue` and use native browser tooltips (with the title attribute instead). Usages should be easy to find:

```
components/Menu/EmojiPickerAction.vue
27:		<NcButton v-tooltip="actionEntry.label"

components/Editor/GuestNameDialog.vue
24:	<form v-tooltip="t('text', 'Enter your name so other people can see who is editing')" class="guest-name-dialog" @submit.prevent="setGuestName()">
49:		tooltip: Tooltip,

components/Editor/AvatarWrapper.vue
8:			:disable-tooltip="true"

components/Editor/SessionList.vue
26:			v-tooltip.bottom="label"
80:		tooltip: Tooltip,

components/Menu/ActionSingle.vue
101:			tooltip,
107:		// do not use tooltip if is a item of action list
111:				name: 'tooltip',
112:				value: tooltip,

components/Menu/ActionList.vue
24:	<NcActions v-tooltip="tooltip"

components/Editor/Status.vue
25:		<div v-tooltip="lastSavedStatusTooltip" class="save-status" :class="saveStatusClass">

components/Menu/BaseActionEntry.js
68:		tooltip() {
```

Also fixes https://github.com/nextcloud/text/issues/4234
